standalone/pherry/src/main.rs

```diff
@@ -915,8 +928,8 @@ async fn bridge(args: Args) -> Result<()> {
         .await?;
 
         // check if pRuntime has already reached the chain tip.
-        if synced_blocks == 0 {
-            if !initial_sync_finished {
+        if synced_blocks == 0 && !more_blocks {
+            if !initial_sync_finished && !args.no_write_back {
```

Comment:
Might need a new option `no_register`

Reply:
Let's rename `no_write_back` to `no_msg_submit`, and add `no_register`?